RELEASE.md

```diff
@@ -0,0 +1,83 @@
+# Release Checklist
+
+The following checks should be completed before releasing a new version of the
+Polkadot/Kusama/Westend runtime or client:
+
+### All Releases
+
+- [ ] Check that the new client releases have [run on the network](#burn-in) without issue for 24
+  hours.
+- [ ] Add [build artifacts](#build-artifacts) to the release.
+
+### Runtime Releases
+
+- [ ] List any [native runtime](#native-runtimes) versions associated with the release.
+- [ ] Increment [`spec_version`](#spec-version) for any native runtimes.
+- [ ] Verify [new migrations](#new-migrations) complete successfully, and the runtime state is
+  correctly updated.
+- [ ] Verify previously [completed migrations](#old-migrations-removed) are removed.
+- [ ] Verify pallet and [extrinsic ordering](#extrinsic-ordering) has stayed the same. Bump
+  `transaction_version` if not.
+- [ ] Verify new extrinsics have been correctly whitelisted/blacklisted for
+  [proxy filters](#proxy-filtering).
+- [ ] Verify [benchmarks](#benchmarks) have been updated for any modified runtime logic.
+- [ ] Verify [Polkadot JS API](#polkadot-js) are up to date with the latest runtime changes.
+
+## Notes
+
+### Burn In
+
+Ensure that Parity DevOps has run the new release on Westend, Kusama, and Polkadot validators for
+at least 24 hours prior to publishing the release.
+
+### Build Artifacts
+
+Add any necessary assets to the release. They should include:
+
+- Linux binary
+- GPG signature
+- SHA256 of binary
+- Source code
+- Wasm binaries of any runtimes
+
+### Native Runtimes
+
+The release notes should list which native runtimes and their versions are included.
```

Comment:
```suggestion
### Release notes

The release notes should list:

- The priority of the release (i.e., how quickly users should upgrade)
- Which native runtimes and their versions are included
- The proposal hashes of the runtimes as built with [srtool](https://gitlab.com/chevdor/srtool)

The release notes may also list:

- Free text at the beginning of the notes mentioning anything important regarding this release
- Notable changes (those labelled with B[1-9]-* labels) separated into sections
- 
```

Expanded this section a bit to codify the layout of the release notes in general.